samples/example/README.md

```diff
@@ -0,0 +1,30 @@
+# Nixer Plugin Example Application
+
+Example Spring Boot web application demonstrating how to use Nixer Spring Plugin.
+
+The application also serves as a reference implementation used in our integration tests,
+which guarantees it is up to date with the latest version of the plugin.
+
+This is a classic _Todo app_ so it consists of only authentication logic and a basic fronted.
```

Comment:
This is not a todo app. It's just a login-logout app. Or am I missing something?

Reply:
I consider "todo app" as an application skeleton (so including auth) with no business logic, used ofter for tutorials or technology demos, etc. 
Do you see this name as something different? I am ok with changing this to something different if you'd like.

Reply:
I am familiar with "todo app" from frontend world, where it means an app that lets you add and remove tasks from a list. For example: http://todomvc.com/examples/backbone/

So I think it wil be better to just call it sample app or similarly.

Reply:
Ok, I wasn't aware it is a more grounded term. I changed this naming.